modules/@apostrophecms/area/index.js

```diff
@@ -763,8 +741,7 @@ module.exports = {
           widgetPreview,
           contextualWidgetDefaultData,
           widgetManagers,
-          action: self.action,
-          widgetOperations
+          action: self.action
```

Comment:
No in area anymore, but in each widget config

Reply:
You need to remove `self.widgetOperations = [];` in the `init` method